go/ct/common/revisions.go

```diff
@@ -11,110 +11,38 @@
 package common
 
 import (
-	"encoding/json"
 	"fmt"
 	"math"
-	"regexp"
-)
-
-type Revision int
 
-const (
-	R07_Istanbul Revision = iota
-	R09_Berlin
-	R10_London
-	R11_Paris
-	R12_Shanghai
-	R13_Cancun
-	R99_UnknownNextRevision
+	"github.com/Fantom-foundation/Tosca/go/tosca"
 )
 
-// Newest Revision currently supported by the CT specification
-const NewestSupportedRevision = R13_Cancun
-const NewestFullySupportedRevision = R12_Shanghai
+// Newest tosca.Revision currently supported by the CT specification
+const NewestSupportedRevision = tosca.R13_Cancun
+const NewestFullySupportedRevision = tosca.R12_Shanghai
 
-const MinRevision = R07_Istanbul
+const R99_UnknownNextRevision = NewestSupportedRevision + 1
```

Comment:
Since the revision is stored in JSON files as part of regression tests, this should be a constant (also over time). Thus, when new revisions are added, this should not collide with new revision numbers nor being altered.

I'd suggest to assign it the value 99, as it is prefixed with.